src/mk_links.sh

```diff
@@ -23,7 +26,8 @@ recursive_setup() {
 			ln -sf $file $home_twin #overwrite incorrect links
 		elif [ -f $file ] && [ ! -h $home_twin ]; then
 			if [ -f $home_twin ]; then
-				printf "The file $file already exists. Overwrite it? (y=yes n=no a=archive) "
+				e_warning "The file $file already exists."
+				e_prompt "Overwrite it? (y=yes n=no a=archive)"
```

Comment:
I have not tried it but I think it will appear as "Try again, shall I overwrite it ? (y=yes n=no a=archive)?", that is, with one too many exclamation marks.